python/examples/integrations/Dask_Profiling.ipynb

```diff
@@ -1555,7 +1555,7 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3.9.13 ('.venv': poetry)",
+   "display_name": "Python 3 (ipykernel)",
```

Comment:
@jamie256 same as another comment, is this something we want changed for everyone?